src/secret-storage.ts

```diff
@@ -100,10 +100,12 @@ export interface PassphraseInfo {
  * Options for {@link ServerSideSecretStorageImpl#addKey}.
  */
 export interface AddSecretStorageKeyOpts {
-    pubkey?: string;
+    /** Information for deriving the key from a passphrase if any. */
     passphrase?: PassphraseInfo;
+    /** Optional name of the key. */
     name?: string;
-    key?: Uint8Array;
+    /** The private key. Will be used to generate the key check values in the key info, it will not be stored on the server */
```

Comment:
Nit:

```suggestion
    /** The private key. Will be used to generate the key check values in the key info; it will not be stored on the server */
```